lua/core/menu/params.lua

```diff
@@ -198,9 +203,8 @@ m.key = function(n,z)
         m.mpos = 1
         m.mode = mMAPEDIT
         m.pm = pm
-      else
-        m.fine = true
       end
+      m.fine = true
     elseif n==3 and z==0 then
```

Comment:
this seems unintentional (we don't want to enter fine mode for all cases)

Reply:
line 201-202 are supposed to stay, but perhaps they should be made more explicit (this case should happen on number and control types, but we're just doing a catch-remaining else)

lines 204-205 (key 3 up) turn off fine

Reply:
but correct, line 207 was wrong